scripts/o3de/ExportScripts/IgnoreAssetProcessor.profile.setregpatch

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
A few other PRs made it into this one :P

Reply:
yea, my PR has this already:
https://github.com/o3de/o3de/pull/17238/files